runners/java-fn-execution/src/main/java/org/apache/beam/runners/fnexecution/control/SdkHarnessClient.java

```diff
@@ -141,7 +171,11 @@ private BundleProcessor(
           fnApiDataService.send(
               LogicalEndpoint.of(bundleId, remoteInput.getTarget()), remoteInput.getCoder());
 
-      return new ActiveBundle(bundleId, specificResponse, dataReceiver, outputClients);
+      return new ActiveBundle(bundleId,
```

Comment:
there is some inconsistency here as to whether or not to place a line break before the first parameter of a long invocation (see lines 343, 396, 434).  I know I have my own preference (in favor of the line break), but I don't know if we have an explicit style convention around it.  Are you aware of any?

Reply:
Not that I'm aware of. Fixed up the few nits that you recommended and any others that I could see.